### ISSUE TITLE
Vue template syntax errors

### ISSUE THREAD

**Issue by gedgud** [w33ble/customer-manager-vue-demo#2]
```
ERROR in ./~/vue-loader/lib/template-compiler.js?id=data-v-6e86ad4b!./~/vue-loader/lib/selector.js?type=template&index=0!./src/pages/Customers.vue
(Emitted value instead of an instance of Error) 
  Vue template syntax error:

  tag <customer-card> has no matching end tag.

 @ ./src/pages/Customers.vue 9:2-166
 @ ./src/router/index.js
 @ ./src/main.js
 @ multi ./src/main.js webpack-hot-middleware/client
ERROR in ./~/vue-loader/lib/template-compiler.js?id=data-v-6e86ad4b!./~/vue-loader/lib/selector.js?type=template&index=0!./src/pages/Customers.vue
(Emitted value instead of an instance of Error) 
  Vue template syntax error:

  tag <customer-row> has no matching end tag.

 @ ./src/pages/Customers.vue 9:2-166
 @ ./src/router/index.js
 @ ./src/main.js
 @ multi ./src/main.js webpack-hot-middleware/client
```
Versions:
Vue.js v2.2.6
NodeJS v6.9.4

**Comment by vishwasrao** [comment]
Hi,
I have fixed it. Check PR.
Thanks,
Vishwas

**Comment by w33ble** [comment]
Closed by #1
